src/components/App/index.tsx

```diff
@@ -36,23 +37,45 @@ const ProjectsPage = React.lazy(() =>
 type ModiteListTypes = 'globe' | 'list'
 
 const Inner = () => {
+//  const listOptions = JSON.parse(localStorage.getItem("list-options")) || {
+//    view: "list",
+//    sort: "lasta",
+//  }
   const [state, dispatch]: DataContextArray = useData()
   const lastLocation: LastLocationType = useLastLocation()
   const location = useLocation()
   const [viewport, setViewport]: MapContextArray = useMap()
   const go = useNavigate('/{tab}')
   const [isLoaded, setIsLoaded] = useState(false)
   const [moditeListType, setModiteListType] = useState<ModiteListTypes>('list')
+  const [listOptionsPopover, setListOptionsPopover] = useState(false)
   const [searchBarCollapsed, setSearchBarCollapsed] = useState(false)
   const activePage = locationToViewType(location.pathname)
   const isModite = activePage === VIEW_TYPES.modite
   const isProject = activePage === VIEW_TYPES.project
   const isProjects = activePage === VIEW_TYPES.projects
   const isTeam = activePage === VIEW_TYPES.modites
-  const isGlobe = moditeListType === 'globe'
+//  let isGlobe = listOptions.view === 'globe'
+  let isGlobe
   const showTabBar = isLoaded && (isProjects || (isTeam && !isGlobe)) && !isProject
 
+  const updateGlobe = () => {
+    const listOptions = JSON.parse(localStorage.getItem("list-options")) || {
```

Comment:
since localStorage can error out on some systems (browsers) when the user is in private mode, I usually recommend to work with browser storage from an abstracted function that would check that loclastorage is available and perhaps even try/catch

Reply:
in src/index.tsx

![2021-04-01 at 7 10 AM](https://user-images.githubusercontent.com/329637/113306893-68d00380-92b9-11eb-8368-0d1d3b2db871.png)

I'm leaving it as-is there.